helm_deploy/dps-data-compliance/templates/housekeeping-cronjob.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: batch/v1 CronJob
+apiVersion: batch/v1beta1
```

Comment:
Attempted to update as we have the following warning, however it is unavailable.

Warning message:
 `batch/v1beta1 CronJob is deprecated in v1.21+, unavailable in v1.25+; use batch/v1 CronJob`